tests/lib/RichText/TextExtractor/ShortTextExtractorTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Tests\FieldTypeRichText\RichText\TextExtractor;
+
+use Ibexa\FieldTypeRichText\RichText\TextExtractor\ShortTextExtractor;
+
+final class ShortTextExtractorTest extends BaseTest
+{
+    protected function setUp(): void
+    {
+        $this->textExtractor = new ShortTextExtractor();
+    }
+
+    public function providerForTestExtractText(): array
+    {
+        return [
+            'simple document' => [
+                $this->getSimpleDocBookXml(),
+                'Welcome to Ibexa',
+            ],
+            'empty xml' => [
+                $this->getEmptyXml(),
+                '',
+            ],
+        ];
+    }
+
+    private function getSimpleDocBookXml(): string
+    {
+        return <<<XML
+<?xml version="1.0" encoding="UTF-8"?>
+<section xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:ezxhtml="https://ezplatform.com/xmlns/docbook/xhtml">
+  <title ezxhtml:level="2">Welcome to Ibexa</title>
+  <para><link xlink:href="ezurl://1" xlink:show="none">Ibexa</link> is the new generation DXP from Ibexa.</para>
+</section>
+XML;
```

Comment:
Heredoc most likely accepts intendation.

Reply:
![image](https://github.com/ibexa/fieldtype-richtext/assets/10212760/4c5b26e9-ac52-4af8-954a-123a359bca72)

PHPStorm reports it as invalid syntax.

Reply:
Hm, that's weird. Regardless, we use that already e.g. in Product Catalog: https://github.com/ibexa/product-catalog/blob/main/tests/integration/REST/ProductQueryWebTest.php#L65-L80. This isn't a blocker though, so to me, it can stay as-is.

Reply:
Looks like it doesn't like it when there is whitespace before `<?xml .. ?>` although semantically it's a valid heredoc. I'll leave it as it to save the world from complaints there are red markings in PHP Storm.